src/Sparrow.Server/Platform/Posix/CGroupHelper.cs

```diff
@@ -152,7 +155,18 @@ private static bool CheckLimitValues(string textValue, out long? value)
         }
     }
 
-    protected abstract string FindCGroupPathForMemory();
+    private string FindCGroupPathForMemory()
+    {
+        if (_memoryDisabled)
```

Comment:
```suggestion
        if (_missingControllerGroups)
```

Reply:
Since for now we only have the "memory" controller group and the field is private I preferred to explicitly show that there is no other controller group in use.
But I changed it now